### PR TITLE
Fix BufferLoad=False with stream-k

### PR DESCRIPTION
**Pull request by AlexBrownAMD**
Added new CI test cases for it as well

### CODE REVIEW

File: Tensile/KernelWriterAssembly.py [hunk 1]
```diff
@@ -5140,8 +5140,24 @@ def graAddresses(self, kernel, tP, isPap=False):
       #kStr += self.bomb(0x13) # after addresses and SRD set
     else:
       tmp = self.vgprPool.checkOut(2, "tmp", self.preventVgprOverflowDuringNewTile)
-      kStr += inst("v_mov_b32", vgpr(tmp+0), sgpr("Address%s+0"%tP["tensorChar"]), "" )
-      kStr += inst("v_mov_b32", vgpr(tmp+1), sgpr("Address%s+1"%tP["tensorChar"]), "" )
+            
+      if kernel["StreamK"]:
+        # StreamK partial tile - offset to tile start index
+        tmpOffset = self.sgprPool.checkOut(2, "skStartOffset", preventOverflow=0)
```
Comment (nakajee) [inline Tensile/KernelWriterAssembly.py:5146]: Probably, we can simply remove tmpOffset and use tmp instead.
Then, we do not need v_mov_b32 to copy from tmpOffset to tmp.

Comment (nakajee) [reply Tensile/KernelWriterAssembly.py:5146]: sorry. This is sgpr...

**Review by nakajee — APPROVED** [review]
Looks good if all tests pass

**Comment by AlexBrownAMD** [comment]
One test failure on gfx90a, seems unrelated and unable to repro error locally. Going to rerun those tests.

**Review by bethune-bryant — APPROVED** [review]

**Pull request merged**